README.md

```diff
@@ -1,15 +1,5 @@
 # Jbuilder
 
-[![Build Status](https://api.travis-ci.org/rails/jbuilder.svg)][travis]
-[![Gem Version](http://img.shields.io/gem/v/jbuilder.svg)][gem]
-[![Code Climate](http://img.shields.io/codeclimate/github/rails/jbuilder.svg)][codeclimate]
-[![Dependencies Status](http://img.shields.io/gemnasium/rails/jbuilder.svg)][gemnasium]
-
-[travis]: https://travis-ci.org/rails/jbuilder
-[gem]: https://rubygems.org/gems/jbuilder
-[codeclimate]: https://codeclimate.com/github/rails/jbuilder
-[gemnasium]: https://gemnasium.com/rails/jbuilder
-
```

Comment:
Not sure I'm fully onboard with removing badges from README. What's the rationale behind this?